src/i18n/en/nav.ts

```diff
@@ -30,13 +30,13 @@ export default [
 	{ text: 'Routing', slug: 'core-concepts/routing', key: 'core-concepts/routing' },
 	{ text: 'Static Assets', slug: 'guides/imports', key: 'guides/imports' },
 	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
+	{ text: 'Deploy', slug: 'deploy/netlify', key: 'guides/deploy' },
```

Comment:
The slug is how we've dealt with the "install" pages, too. Right now we have a folder `/install/` which contains `auto.md` and `manual.md`. 

I used the same pattern so that the Markdown pages for `netlify.md` and `vercel.md` and `other.md` (which may be split out further) are located in a `/deploy/` folder.